src/test/java/com/ksm/wstbackoffice/controller/CountryControllerIT.java

```diff
@@ -37,6 +40,24 @@
         );
     }
 
+    public static Stream<Arguments> findAllByTestArguments() {
+        return Stream.of(
+                Arguments.of(Arrays.asList(), 400),
+                Arguments.of(Arrays.asList(""), 400),
+                Arguments.of(Arrays.asList("a", "ua", "b"), 200)
```

Comment:
Why do you put the positive test in the negative test case?

Reply:
fixed